src/Sylius/Bundle/ResourceBundle/DependencyInjection/Configuration.php

```diff
@@ -58,28 +57,58 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('object_manager')->defaultValue('default')->end()
                             ->scalarNode('templates')->cannotBeEmpty()->end()
                             ->arrayNode('classes')
+                                ->isRequired()
+                                ->addDefaultsIfNotSet()
                                 ->children()
                                     ->scalarNode('model')->isRequired()->cannotBeEmpty()->end()
-                                    ->scalarNode('controller')->defaultValue('Sylius\Bundle\ResourceBundle\Controller\ResourceController')->end()
-                                    ->scalarNode('repository')->end()
+                                    ->scalarNode('interface')->cannotBeEmpty()->end()
+                                    ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
+                                    ->scalarNode('repository')->cannotBeEmpty()->end()
                                     ->scalarNode('factory')->defaultValue(Factory::class)->end()
-                                    ->scalarNode('interface')->end()
-                                    ->arrayNode('translation')
+                                    ->arrayNode('form')
+                                        ->prototype('scalar')->end()
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('validation_groups')
+                                ->addDefaultsIfNotSet()
+                                ->children()
+                                    ->arrayNode('default')
+                                        ->prototype('scalar')->end()
+                                        ->defaultValue(array())
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('translation')
+                                ->addDefaultsIfNotSet()
```

Comment:
this seems wrong to me. you cannot disable translations per resource with this. @pjedrzejewski